S/SuiteSparse/SuiteSparse_GPU/build_tarballs.jl

```diff
@@ -87,12 +87,13 @@ append!(products, [
     LibraryProduct("libSuiteSparse_GPURuntime", :libSuiteSparse_GPURuntime),
 ])
 
-append!(dependencies, [
-    Dependency("METIS_jll"),
-    Dependency("MPFR_jll"),
-    Dependency("GMP_jll"),
+dependencies = [
+    dependencies;
+    Dependency("METIS_jll")
+    Dependency("MPFR_jll")
+    Dependency("GMP_jll")
     BuildDependency(PackageSpec(name="CUDA_full_jll", version=v"10.0"))
```

Comment:
I suppose it does not like `v"10.0"`.

Reply:
I think you need to specify one of the actual [registered versions](https://github.com/JuliaRegistries/General/blob/46718fe08a481d82ba7b5ed9b9a0154f8f9641ca/C/CUDA_full_jll/Versions.toml) (excluding the build number).  So for example `v"10.0.130"`

Reply:
Indeed: this is not like the "compat" field in a Project.toml, it specifies an *exact* version to be used. If you want a range, you can assign a suitable `VersionSpec` instance here.